lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class PrometheusMetrics {
+  constructor (circuit) {
+    this.circuit = circuit;
+    // this.client = require('prom-client');
+    this.collect();
+  }
+
+  collect (prefix) {
+    this.collectDefaultMetrics = this.client.collectDefaultMetrics({ prefix: prefix, timeout: 5000 });
+    this.circuit.on('open', () => {
+      this.clear();
+    });
+    this.circuit.on('halfOpen', () => {
+      this.clear();
+    });
+  }
+
+  clear () {
+    clearInterval(this.client.collectDefaultMetrics());
```

Comment:
Use `this.interval` here. Otherwise, you're just starting a new thing that will start counting.

```suggestion
    clearInterval(this.interval);
```